src/lib/db/query.js

```diff
@@ -3,30 +3,33 @@
 const db = require('./postgres');
 
 const timeZone = 'AT TIME ZONE \'europe/london\'';
-const countsByType = 'SELECT dataset, count(*)::INTEGER FROM lev_audit WHERE date_time > $(from)';
+const DTZ = `(date_time ${timeZone})::DATE`;
+const countsByType = `SELECT dataset, count(*)::INTEGER FROM lev_audit WHERE ${DTZ} > $(from)`;
 const countsByUser =
-  'SELECT date_time::DATE AS date, dataset, username, count(*)::INTEGER FROM lev_audit WHERE date_time > $(from)';
-const until = 'AND date_time < $(to)';
+  `SELECT ${DTZ} AS date, dataset, username, count(*)::INTEGER 
+  FROM lev_audit WHERE ${DTZ} > $(from)`;
```

Comment:
Any reason this query is split over 2 lines?

Reply:
Was because of linting error but after refactor, no. Have changed.